packages/flame/lib/src/parallax.dart

```diff
@@ -196,9 +197,8 @@ class ParallaxAnimation extends ParallaxRenderer {
 
     final animation =
         await SpriteAnimation.load(path, animationData, images: images);
-    final prerenderedFrames = await Future.wait(
-      animation.frames.map((frame) => frame.sprite.toImage()).toList(),
-    );
+    final prerenderedFrames =
+        animation.frames.map((frame) => frame.sprite.toImageSync()).toList();
```

Comment:
When we are in an async context, why would we want to use the sync method? This also loads them in a serial manner instead of parallelized like the old method did.

Reply:
Hmm, I can also create here additional sync constructor. 
According to docs, there is no problem with 'serial manner' because rasterizing going async under hood. But all exceptions will be thrown at the render time, not in toImageSync function.

Reply:
Done.